ReportPortal.Tests/ReportPortal.E2E.Core/StatusCodes/ResponseStatus.cs

```diff
@@ -0,0 +1,8 @@
+﻿namespace ReportPortal.E2E.API.Business.Models
+{
+    public enum ResponseStatus
+    {
+        BadRequest = 4001,
+        NotFound = 4041
+    }
+}
```

Comment:
May be you can use System.Net ( public enum HttpStatusCode)?

Reply:
Internal report portal response codes are little bit different then usual codes. As you could see 4001 and standard bad request code is 400. So it is impossible to use HttpStatusCode in this case